common/src/service/HttpClientHelper.cs

```diff
@@ -289,12 +269,8 @@ internal sealed class HttpClientHelper : IHttpClientHelper
                 {
                     // TODO: skintali: Use string etag when service side changes are ready
```

Comment:
Is this still a todo? 
May be not in scope of your PR but wondering since requestMsg is used below

Reply:
I am not sure what this TODO even refers to, maybe something to take up outside the scope of this PR.